src/edge_utils.c

```diff
@@ -2004,6 +2020,13 @@ void edge_send_packet2net (n2n_edge_t * eee,
     /* Once processed, send to destination in PACKET */
 
     memcpy(destMac, tap_pkt, N2N_MAC_SIZE); /* dest MAC is first in ethernet header */
+    if ((eee->conf.allow_routing) && (!is_multi_broadcast(destMac))) {//find the destMac behind which edge, and change dest to this edge
```

Comment:
probably a linefeed missing in front of the comment... :monocle_face:    :wink:

Reply:
yeah, I will move the comment to a new line.